### DISCUSSION TITLE
ADB over ethernet?

### DISCUSSION THREAD

**Discussion by JackCA** [iamromulan/cellular-modem-wiki#3] [Help - Quectel] [answered]
I have my install in a very difficult to reach location -- is there any way to enable ADB over the ethernet port, rather than needing to be directly connected via USB?

**Comment by iamromulan** [comment, marked as answer]
> I have my install in a very difficult to reach location -- is there any way to enable ADB over the ethernet port, rather than needing to be directly connected via USB?

Yes! It's not ADB exactly but it works much much better.

It's called ttyd, installable through the toolkit 😁

I am also looking into setting up an ftp server as well so you can access the modems filesystem from FileZilla or another ftp client (even windows file explorer)

**Reply by JackCA** [reply]
Okay, looks like I'll make one last trip and turn that feature on :)

**Reply by iamromulan** [reply]
> Okay, looks like I'll make one last trip and turn that feature on :) 

To access it
In your web browser,
Go to http://192.168.225.1:443/
Username root
Password is whatever you set it as during the installation process
